prow/plugins.yaml

```diff
@@ -357,6 +357,20 @@ require_matching_label:
 
     Note: Method 1 will trigger an email to the group. See the [group list](https://git.k8s.io/community/sig-list.md).
     The `<group-suffix>` in method 1 has to be replaced with one of these: _**bugs, feature-requests, pr-reviews, test-failures, proposals**_.
+# kubernetes/test-infra wants needs-sig on issues only, and is piloting
```

Comment:
I do not want to require sig-testing on test-infra issues, feel like unnecessary noise to me.

I would prefer we just close out non sig-testing issues, helping people file it in a more appropriate location.

Reply:
The thing is there are a number of config related issues in this repo, which aren't necessarily out of place, they're just not on sig-testing to address because they can be handled via self-service (eg: release job munging, scalability configs, contributor experience stuff)

If I'm the only one who thinks it's useful to see `org:kubernetes label:sig/testing` issues then I'll let it drop

Reply:
I'm okay with this change fwiw (even redundant labels)

Reply:
I'd really like to expand the gubernator PR dashboard to provide an issue dashboard. Until then I don't think labels like this are extremely useful for most people.

I'm also fine with the change (hence the LGTM)